src/app/shared/enums/config.ts

```diff
@@ -26,6 +26,13 @@ export class ConfigKeys {
   static APP_CREDITS_TITLE = new ConfigKeys('app_credits_title')
   static APP_CREDITS_BODY = new ConfigKeys('app_credits_body')
 
+  static QUESTIONNAIRE_NEXT_BUTTON_AUTOMATIC_SET = new ConfigKeys(
+    'questionnaire_next_button_automatic_set'
+  )
+  static QUESTIONNAIRE_NEXT_BUTTON_ENABLED_SET = new ConfigKeys(
+    'questionnaire_next_button_enabled_set'
```

Comment:
Sorry forgot to mention before, but would be nice to have similar names in the remote config too for easy configuration.